src/Preflight/PreflightArgs.php

```diff
@@ -227,6 +231,11 @@ public function selectedSite($default = false)
         return $this->get(self::ROOT, $default);
     }
 
+    public function setDebug($value)
+    {
+        $this->set(self::DEBUG, $value);
```

Comment:
Sort of wondering if we need to set the debug flag in the pass through args too.

Reply:
The --debug option is declared in Application.php so nothing more to do, if I understand your comment correctly.

Reply:
Yes, `--debug` is defined as a global option; however, if you run `drush status --debug`, it no longer turns on debug mode for most of Drush. The value is only set for preflight.